samples/MQL4/MovingAverages.mqh

```diff
@@ -0,0 +1,240 @@
+//+------------------------------------------------------------------+
+//|                                               MovingAverages.mqh |
+//|                   Copyright 2009-2013, MetaQuotes Software Corp. |

```

Comment:
This particular file is publicly available (https://www.mql5.com/en/code/7765), however these files have been released without any license information.
I assume, it would be better to provide other samples instead?